pbs_light.go

```diff
@@ -44,16 +44,18 @@ type AccountMetrics struct {
 	RequestMeter      metrics.Meter
 	BidsReceivedMeter metrics.Meter
 	PriceHistogram    metrics.Histogram
+	AdapterMetrics    map[string]*AdapterMetrics
```

Comment:
Would recommend `map[Bidder]*adapterMetrics` here. Stringly-typed variables are generally a bad idea in strictly typed languages.

...unless one of the people more familiar with Go thinks the performance gain of using a `string` is worth it. Then at least leave a comment here saying that the `string` is really a `Bid.bidderCode`